packages/sitecore-jss-tracking/src/dataModels.ts

```diff
@@ -33,3 +33,20 @@ export interface PageViewInstance {
   /** URL of page to track - should be absolute route path but can accept any value */
   url: string;
 }
+
+export interface PageViewData {
+  /** URL of page to track - should be absolute route path but can accept any value */
+  url: string;
+
+  /** URL of the page that loaded the trackable page */
```

Comment:
Not sure that "loaded" and "trackable" words are the right one. What about "URL of the page which is linked to the page to track"